### PR TITLE
Scene 5 floors with angles

### PR DESCRIPTION
**Pull request by vavadym**
found a bug with scene creations

### CODE REVIEW

File: concentration/ViewControllers/MenuViewController.swift [hunk 1]
```diff
@@ -8,16 +8,16 @@
 import SpriteKit
 
 struct IASceneType {
-    let scene: SKScene
+    let scene: SKScene.Type
```
Comment (vavadym) [inline concentration/ViewControllers/MenuViewController.swift:11]: this is not right place to change it, we will create new task and branch for his bug

File: concentration/Scene/FloorsScene.swift [hunk 2]
```diff
@@ -9,8 +9,97 @@ import SpriteKit
...
+        
+        self.addChild(leftWall)
+        self.addChild(rightWall)
+        
+        let sqns = SKAction.sequence([
+            .run { self.removeUpperFloor() },
+            .wait(forDuration: 2)
+        ])
+        self.run(.repeatForever(sqns))
+    }
```
Comment (vavadym) [inline concentration/Scene/FloorsScene.swift:107]: need more animations

File: concentration/Scene/FloorsScene.swift [hunk 3]
```diff
@@ -9,8 +9,97 @@ import SpriteKit
...
+        
+        let firstFloor = floors.removeFirst()
+        firstFloor.removeFromParent()
+        
+        if let nextFloor = floors.first {
+            cameraNode.run(.moveTo(y: nextFloor.position.y + 100, duration: 1))
+        }
+    }
+    
+    func setElements() {
```
Comment (miroshniktyt) [inline concentration/Scene/FloorsScene.swift]: need to separate into another class that will do the calculation of frames for different matrixes

**Review by miroshniktyt — CHANGES_REQUESTED** [review]

**Review by miroshniktyt — APPROVED** [review]

**Comment by miroshniktyt** [comment]
good job me

**Pull request closed without merging**